Podfile.lock

```diff
@@ -125,4 +125,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ab6619ebb1d933652ad9c47b1c330909f098fb41
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.14.2
```

Comment:
@SoumenRautray This line should be committed?

Reply:
@anish-munirathinam reverted this to 1.13.0